src/material.h

```diff
@@ -422,7 +406,24 @@ namespace pyne
     /// Returns a mapping of the nuclides in this material to their atom densities.
     /// This calculation is based off of the material's density.
     std::map<int, double> to_atom_dens();
-
+#ifdef PYNE_DECAY
+    /// Calculates the activity of a material based on the composition and each
+    /// nuclide's mass, decay_const, and atmoic_mass.
+    comp_map activity();
```

Comment:
None of these rely on decay, do they?  Just on decay data which is found elsewhere.  Can't we keep these even if decay is not included?

Reply:
we surely can

Reply:
this is correct now only include `Material::decay()` in the `IFDEF`